refs/biblio.json

```diff
@@ -2124,6 +2124,26 @@
         "publisher": "Object Management Group",
         "date": "January 2008"
     },
+    "OPENAPIS": {
+        "href": "https://www.openapis.org/",
+        "title": "OpenAPI Specification",
+        "publisher": "OpenAPI Initiative",
+        "versions": {
+            "3.0.2": {
+                "href": "http://spec.openapis.org/oas/v3.0.2",
+                "title": "OpenAPI Specification - version 3.0.2",
+                "rawDate": "2018-10-08",
+                "authors": [
```

Comment:
You want to share this data (notably authors) with the main entry, too.

Reply:
thanks for the feedback! See my edits.

Author's list is different for every version, I hope the way I defined them is ok (all authors merged into a single array). Regarding dates, since it is a living standard, I do not know if it appropriate to add the date of a specific version.